limitador-server/sandbox/docker-compose-limitador-memory.yaml

```diff
@@ -3,6 +3,9 @@ version: '3.8'
 services:
   limitador:
     image: ${LIMITADOR_IMAGE:-limitador-testing}
+    build:
```

Comment:
and if I do not want to build image and use an existing one instead?

Reply:
feel free to test it out.  by default it should first pull the image if It does not exist and use that.  If it can't pull it, then it builds it.

Reply:
cool stuff. Great contribution.

with empty local docker image repo it builds a new image `limitador-testing`
```
> make deploy-in-memory
....
❯ docker compose -p sandbox ps
NAME                  IMAGE                           COMMAND                  SERVICE     CREATED              STATUS              PORTS
sandbox-envoy-1       envoyproxy/envoy:v1.20-latest   "/docker-entrypoint.…"   envoy       About a minute ago   Up About a minute   10000/tcp, 0.0.0.0:18000->80/tcp, :::18000->80/tcp, 0.0.0.0:18001->8001/tcp, :::18001->8001/tcp
sandbox-limitador-1   limitador-testing               "limitador-server --…"   limitador   About a minute ago   Up About a minute   0.0.0.0:18080->8080/tcp, :::18080->8080/tcp, 0.0.0.0:18081->8081/tcp, :::18081->8081/tcp
sandbox-upstream-1    kennethreitz/httpbin            "gunicorn -b 0.0.0.0…"   upstream    About a minute ago   Up About a minute   80/tcp
```

When I want to use `quay.io/kuadrant/limitador:latest`, I can easily run it.
```
> make deploy-in-memory LIMITADOR_IMAGE=quay.io/kuadrant/limitador:latest
....
❯  docker compose -p sandbox ps
NAME                  IMAGE                               COMMAND                  SERVICE     CREATED              STATUS              PORTS
sandbox-envoy-1       envoyproxy/envoy:v1.20-latest       "/docker-entrypoint.…"   envoy       About a minute ago   Up About a minute   10000/tcp, 0.0.0.0:18000->80/tcp, :::18000->80/tcp, 0.0.0.0:18001->8001/tcp, :::18001->8001/tcp
sandbox-limitador-1   quay.io/kuadrant/limitador:latest   "limitador-server --…"   limitador   About a minute ago   Up About a minute   0.0.0.0:18080->8080/tcp, :::18080->8080/tcp, 0.0.0.0:18081->8081/tcp, :::18081->8081/tcp
sandbox-upstream-1    kennethreitz/httpbin                "gunicorn -b 0.0.0.0…"   upstream    About a minute ago   Up About a minute   80/tcp
```